cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -299,13 +299,17 @@ public RoleBinding generateRoleBinding(String namespace, String watchedNamespace
                 .withNewMetadata()
                     .withName(roleBindingName(cluster))
                     .withNamespace(watchedNamespace)
-                    .withOwnerReferences(createOwnerReference())
                     .withLabels(labels.toMap())
                 .endMetadata()
                 .withRoleRef(roleRef)
                 .withSubjects(singletonList(ks))
                 .build();
 
+        // We set OwnerReference only within the same namespace since it does nto work cross-namespace
+        if (namespace.equals(watchedNamespace)) {
+            rb.getMetadata().setOwnerReferences(singletonList(createOwnerReference()));
+        }
+
```

Comment:
Do we need this for the (deprecated) `spec.topicOperator` at well?

Reply:
No, that exists now only in the API. But not in the operator modules.